assets/html/documenter.css

```diff
@@ -105,6 +105,20 @@ pre code {
     background-color: initial;
 }
 
+kbd {
+    font: 11px "SFMono-Regular", Consolas, "Liberation Mono", Menlo, Courier, monospace;
```

Comment:
Font should probably be consistent with the other monospace stuff. Instead of having it here we could add `kbd` to the the `pre` and `code` on line 33. Need to make sure it works with the rest of the styling though.